packages/core/src/alert/alert-group.element.ts

```diff
@@ -154,6 +100,19 @@ export class CdsAlertGroup extends LitElement {
     `;
   }
 
+  async updated(props: Map<string, any>) {
+    super.updated(props);
+    await Promise.all(Array.from(this.alerts).map(a => a.updateComplete));
+
+    Array.from(this.alerts).forEach(alert =>
+      syncProps(alert, this, {
+        status: props.has('status') && this.type !== 'light' && alert.status !== 'loading',
+        type: props.has('type'),
+        size: props.has('size'),
+      })
+    );
```

Comment:
Added a util func similar to the one I used in forms. It lets us define a set of common props between a parent and child component and the condition when it should set them.